sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsClient.java

```diff
@@ -494,32 +506,32 @@ public ModelData getModel(String modelId) {
      * Gets a model, including the model metadata and the model definition.
      * @param modelId The Id of the model.
      * @param context Additional context that is passed through the Http pipeline during the service call.
-     * @return The ModelData and the http response
+     * @return A {@link Response} containing a {@link ModelData} instance that contains the model and its metadata.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Response<ModelData> getModelWithResponse(String modelId, Context context) {
         return digitalTwinsAsyncClient.getModelWithResponse(modelId, context).block();
     }
 
     /**
-     * Gets the list of models by iterating through a collection.
```

Comment:
The changes from lines 505 to 522 are just swapping the order that these two list APIs appear. Now the minimal API is above the maximal API for consistency